scripts/scripts.js

```diff
@@ -142,19 +142,23 @@ export function decorateMain(main) {
   decorateVideoLinks(main);
 }
 
-// if it gets a block as argument, treat that as specific styling
-// only for that block
-// otherweise style the entire body
-export function decorateTemplateAndTheme(block = document.body) {
+export function decorateBlockMode(block) {
```

Comment:
What's the point of this wrapper?

Reply:
we have a decorator for template and theme  called `decorateTemplateAndTheme` and we have a decorator for blocks `decorateBlockMode` They both use the same underlying method for decorating different elements. Its semantics.

Similar logic to having domEL and a node and div node and all the others.